core/quorum_initiator.go

```diff
@@ -334,6 +348,11 @@ func (c *Core) initiateConsensus(cr *ConensusRequest, sc *contract.Contract, dc
 		reqPledgeTokens = 1
 	case SmartContractExecuteMode, NFTExecuteMode:
 		reqPledgeTokens = sc.GetTotalRBTs()
+	case FTTransferMode:
```

Comment:
What is the difference between this and reqPledgeTokens in  `case RBTTransferMode, NFTSaleContractMode, SelfTransferMode, PinningServiceMode`: ?

Reply:
obtaining `reqPledgeTokens` is similar to RBTTransferMode, NFTSaleContractMode, SelfTransferMode, PinningServiceMode. Just added as a case, no difference. Can be added with `RBTTransferMode` case also.

Reply:
This switch case scope is to only to obtain `reqPledgeTokens` .

Reply:
Also , majority of code of FT and RBT (part RBT) are same. I would prefer a generalised function to handle both . And we do minimal token based operations within.

Reply:
getting the total token value to be transferred is same for FT transfer and RBT transfer. Also the steps involved in the FT transfer is very same to RBT transfer.